presto-verifier/src/main/java/com/facebook/presto/verifier/framework/ExtendedVerification.java

```diff
@@ -96,19 +100,42 @@ public DataMatchResult verify(
             // Extended verification doesn't support query bank mode for now.
             return dataMatchResult;
         }
+        List<Column> controlColumns = getColumns(getHelperAction(), typeManager, control.getObjectName());
+        List<Column> testColumns = getColumns(getHelperAction(), typeManager, test.getObjectName());
 
         // 1. Partition verification
-        List<Column> controlPartitionColumns = null;
-        List<Column> testPartitionColumns = null;
+        DataMatchResult partitionMatchResult = verifyPartition(control, test, controlColumns, testColumns, controlChecksumQueryContext, testChecksumQueryContext);
+        // 2. Bucket verification
+        DataMatchResult bucketMatchResult = verifyBucket(control, test, controlColumns, testColumns, controlChecksumQueryContext, testChecksumQueryContext);
+
+        if (partitionMatchResult != null) {
+            return partitionMatchResult;
+        }
+        if (bucketMatchResult != null) {
```

Comment:
Should it be `if (!partitionMatchResult.isMatched())` and `if (!bucketMatchResult.isMatched())`

Reply:
For testing, were you able to trigger a bucket mismatch? You can manually switch the names of two underlying WS file.

Reply:
I manually tried
when not changing the data "Progress: 1 succeeded, 0 skipped, 0 resolved, 0 failed, 100.00% done"
when changing the data "2024-03-08T16:29:50.330-0800	INFO	main	com.facebook.presto.verifier.framework.VerificationManager	Progress: 0 succeeded, 0 skipped, 0 resolved, 1 failed, 100.00% done"